python/ola/PidStoreTest.py

```diff
@@ -228,45 +228,53 @@ def testPackRanges(self):
     blob = pid.Pack(args, PidStore.RDM_SET)
     self.assertTrue(len(blob) > 1)
 
+    # invalid year (2002 < 2003)
     with self.assertRaises(PidStore.ArgsValidationError):
-      args = ["2000", "6", "20", "20", "20", "20"]
+      args = ["2002", "6", "20", "20", "20", "20"]
       blob = pid.Pack(args, PidStore.RDM_SET)
 
+    # invalid month < 1
     with self.assertRaises(PidStore.ArgsValidationError):
       args = ["2020", "0", "20", "20", "20", "20"]
       blob = pid.Pack(args, PidStore.RDM_SET)
 
+    # invalid month > 12
     with self.assertRaises(PidStore.ArgsValidationError):
       args = ["2020", "13", "20", "20", "20", "20"]
       blob = pid.Pack(args, PidStore.RDM_SET)
 
+    # invalid month > 255
     with self.assertRaises(PidStore.ArgsValidationError):
       args = ["2020", "255", "20", "20", "20", "20"]
```

Comment:
This is >= 255, do you mean to set the test value to 256?

Reply:
Heh, guess that looks silly.  I'll set it to 256.

Reply:
Done.